recipe/meta.yaml

```diff
@@ -1,26 +1,27 @@
 {% set name = "smirnoff-plugins" %}
-{% set version = "0.0.3" %}
+{% set version = "0.0.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/openforcefield/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: fa206061492eea6b1a8189f17d72e0c6c1fe9eb566bd02beae89563df53b5525
+  sha256: d8f59c06d61b913246dabb55cd7d76eaae1c2eb45f8b0e25df851d1a72b92023
 
 build:
   number: 0
   noarch: python
-  script: "{{ PYTHON }} -m pip install . -vv"
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.9
     - pip
   run:
-    - python >=3.6
-    - openff-toolkit-base >=0.10.6,<0.11.0
+    - python >=3.9
+    - openff-toolkit-base >=0.12.1
```

Comment:
Some other versions might work (unless Interchange also requires this) but this version includes all of the plugin-related bugfixes to date